src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -436,9 +430,11 @@ pg_active_universe_history_internal(FunctionCallInfo fcinfo)
     else
       nulls[j++] = true;
 
-    // Top level node id
+    char top_level_node_id[33];
+    uint128_to_char(top_level_node_id, AUHEntryArray[i].top_level_node_id);
```

Comment:
Does this convert it to the tserver node id? At some point, I would like to add the capability to join this table with a tserver list.

Reply:
Yes, the tserver node id is a 16 bytes hex, so it's broken into two 8 byte integers and propagated everywhere, this just converts these two 8 byte integers back to a single 16 byte hex